src/tests/fuzz/onefile.c

```diff
@@ -13,6 +13,10 @@ int main(int argc, char** argv)
     if (argc != 2) {
         return 1;
     }
+#ifdef AFLFUZZ_PERSISTANT_MODE
```

Comment:
this is set by our own `configure.ac`, and in this branch only with `--enable-afl`. Think it would be good to test for it when just using `--enable-fuzztargets` as well.

Reply:
It does show much better speeds
![Screenshot from 2020-03-03 11-46-35](https://user-images.githubusercontent.com/32410611/75768465-b7247300-5d44-11ea-9a99-7eb0c5488845.png)